deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -530,7 +530,7 @@ async def get_versions(request: Request):
                 "SELECT * FROM pulse3d_versions WHERE state != 'deprecated' ORDER BY created_at"
             )
 
-        return [row["version"] for row in rows]
+        return [{"version": row["version"], "state": row["state"]} for row in rows]
```

Comment:
You could update the the query to `SELECT version, state FROM ...` since the other columns aren't used, and then just do `[dict(row) for row in rows]`, but not a big deal